src/actions/get-change-allowed-tokens-events.ts

```diff
@@ -44,7 +44,7 @@ export async function action(block: DecodedLog<ChangeAllowedTokensEvent['returnV
 
             const [token, created] = await db.tokens.findOrCreate({
               where: {
-                address: tokenAddress,
+                address: tokenAddress.toLowerCase(),
```

Comment:
To ensure that there will be no problems, you can turn the column value into lower too, like we are doing here:
https://github.com/layerx-labs/webnetwork/blob/master/pages/api/search/networks/index.ts#L71